source/locale/pt_BR/LC_MESSAGES/using-web-interface.po

```diff
@@ -40,320 +36,111 @@ msgstr "Nível: básico"
-"compose a device must not share an attribute with the same name."
-msgstr ""
-"Note que, quando um modelo é selecionado no painel direito da tela de "
-"criação de dispositivo, todos os atributos são herdados para aquele "
-"dispositivo. É possível adicionar mais de um modelo, tendo em mente que "
-"modelos que compõem o dispositivo não podem compartilhar atributos com o "
-"mesmo nome. "
+msgid "Note that, when you select the template in the right panel at device creation screen, all attributes are inherited from that device. You could add more templates as needed, keeping in mind that templates used to compose a device must not share an attribute with the same name."
+msgstr "Note que, quando um modelo é selecionado no painel direito da tela de criação de dispositivo, todos os atributos são herdados para aquele dispositivo. É possível adicionar mais de um modelo, tendo em mente que modelos que compõem o dispositivo não podem compartilhar atributos com o mesmo nome. "
 
 #: ../../source/using-web-interface.rst:47
-msgid ""
-"As devices are tightly associated to templates, if you want to remove a "
-"template, you should remove all its associated devices first. If such "
-"thing happens the following error message will appear:"
-msgstr ""
-"Os dispositivos são fortemente atrelados aos a modelos. Para remover um "
-"modelo, deve-se remover primeiro todos os dispositivos a ele associados. "
-"Caso contrário, a seguinte mensagem aparecerá:"
+msgid "As devices are tightly associated to templates, if you want to remove a template, you should remove all its associated devices first. If such thing happens the following error message will appear:"
+msgstr "Os dispositivos são fortemente atrelados aos a modelos. Para remover um modelo, deve-se remover primeiro todos os dispositivos a ele associados. Caso contrário, a seguinte mensagem aparecerá:"
 
 #: ../../source/using-web-interface.rst:58
-msgid ""
-"You can add and remove attributes from templates and they will be "
-"immediately available to devices. In case of new attributes being added, "
-"though, you should keep in mind that there must not be any device with "
-"templates which have attributes with same name. If such thing happens, "
-"the following message will appear:"
-msgstr ""
-"É possível adicionar e remover atributos dos modelos, fazendo com que as "
-"alterações sejam imediatamente refletidas nos dispositivos associados. No"
-" caso de novos adição, no entanto, deve-se observar que os atributos dos "
-"modelos que compõem um determinado dispositivo não podem possuir o mesmo "
-"nome. Se isso acontecer, a seguinte mensagem aparecerá: "
+msgid "You can add and remove attributes from templates and they will be immediately available to devices. In case of new attributes being added, though, you should keep in mind that there must not be any device with templates which have attributes with same name. If such thing happens, the following message will appear:"
```

Comment:
It looks like this was done by hand, since there are no line breaks

Reply:
Add line breaks to keep the standard

Reply:
There are more than 30 cases with no line breaks, add line breaks for all? Will take some time to do that.

Reply:
This should be done automatically when editing with poedit or even executing PO generation commands, but if it is not working, leave it at that.

Reply:
Okay, no changes then.